spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactory.java

```diff
@@ -801,6 +801,9 @@ private void logicalClose(ChannelProxy proxy) throws Exception {
 			if (this.target != null && !this.target.isOpen()) {
 				synchronized (targetMonitor) {
 					if (this.target != null && !this.target.isOpen()) {
+						if (this.target instanceof PublisherCallbackChannel) {
+							this.target.close(); // emit nacks if necessary
```

Comment:
Good. Can I just push this two lines of code into `1.5.x`?
Or is there anything to test against?